rex/resource.py

```diff
@@ -1151,10 +1182,88 @@ def _map_file_instances(h5_files):
 
         return h5_map
 
+    def _preflight_check(self):
+        """
+        Check time_index and coordinates accross files
+        """
+        time_index = None
+        lat_lon = None
+
+        bad_files = []
+        for file in self.h5_files:
+            with Resource(file) as f:
+                ti = f.time_index
```

Comment:
I think this will raise an error if `time_index` is not a dataset in file? See line 626 in this file.

Reply:
I completely forgot about that, fixed